depthai_examples/ros2_src/stereo_publisher.cpp

```diff
@@ -140,7 +140,13 @@ int main(int argc, char** argv){
 
     auto calibrationHandler = device.readCalibration();
 
-    dai::rosBridge::ImageConverter converter(deviceName + "_left_camera_optical_frame", true);
+    auto boardName = calibrationHandler.getEepromData().boardName;
+    if (monoHeight > 480 && boardName == "OAK-D-LITE") {
+        monoWidth = 640;
+        monoHeight = 480;
+    }
```

Comment:
this should go in for all the examples that are using mono cameras. 
And the monoResolution setting in this example needs to be added to `stereo_nodelet` too.

Reply:
and same example in `ros1_src` too